templates/my.cnf.erb

```diff
@@ -6,8 +6,10 @@
 <%     v.sort.map do |ki, vi| -%>
 <%       if ki == 'ssl-disable' or (ki =~ /^ssl/ and v['ssl-disable'] == true) -%>
 <%         next %>
-<%       elsif vi == true or vi == '' -%>
+<%       elsif [true, '', 'on'].include?(vi) -%>
 <%=        ki %>
+<%       elsif [false, 'off'].include?(vi) -%>
```

Comment:
Thanks a lot for the first review!!
I guess I'll need to take some more time for this. My change introduces test failures with 'ssl = false'. I did a quick test on this against Ubuntu 14.04 with MySQL 5.5 and couldn't immediatly prove it's of any real use but I really need to test this further. If you don't mind, I'd like to keep this open until I could check this properly.